docs/script-api.md

```diff
@@ -481,6 +481,11 @@ Send input
 ### [`read_input`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=read_input)
 #### Params: `int uid`
 Read input
+### [`read_stolen_input`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=read_stolen_input)
+#### Params: `int uid`
+<summary>
```

Comment:
What's the deal with these? I've seen one before and it seems to [break the markdown](https://github.com/jeremypepper/overlunky/blob/main/docs/script-api.md#read_stolen_input)

Reply:
No idea I just ran the python script :(

Reply:
fixed, it was the <summary> tags in comments visual studio added, I removed it.